### PR TITLE
Added ability SSH via host and alias

### LINKED ISSUE #30: Add ability to connect via alias/host to server
We need to add an ability to use the alias or only host address for ssh connections. My dev should get the username and password from DB

### PR DESCRIPTION
**Pull request by esikachev**
Now we need to connect via "ssh host" if host already in DB.
Close #30

### CODE REVIEW

File: my_dev/config.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 opts = [
     cfg.StrOpt('host',
-               default='http://localhost:5000',
+               default='http://127.0.0.1:5000',
```
Comment (zemuvier) [inline my_dev/config.py]: Why did you change this line?

File: my_dev/tests/functional/base.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (zemuvier) [inline my_dev/tests/functional/base.py]: Remove this line

File: my_dev/tests/functional/base.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import sys
+
+import testtools
+
+from my_dev.tests.functional import utils
+from my_dev import users
+from my_dev import runner
```
Comment (zemuvier) [inline my_dev/tests/functional/base.py]: This should be in the alphabet order

File: my_dev/tests/functional/base.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+from my_dev import ssh
+
+
+class TestBase(testtools.TestCase):
+    def setUp(self):
+        super(TestBase, self).setUp()
+        self.user = users.Users()
+
+    def create_user(self, username=None, password=None, email=None):
+        username = username if username else utils.rand_name('username')
```
Comment (zemuvier) [inline my_dev/tests/functional/base.py]: Why don't you use get-method? 

Other way: create 1 dict var args. Than it can look like this:
_if auth:
     username = auth['username']
 else:
    username = utils.rand_name('username')_

File: my_dev/tests/functional/test_client.py [hunk 5]
```diff
@@ -2,39 +2,32 @@
 
 import testtools
 
+from my_dev.tests.functional import base
 from my_dev.tests.functional import utils
-import my_dev.users as users
+from my_dev import users
```
Comment (zemuvier) [inline my_dev/tests/functional/test_client.py]: ditto

File: my_dev/tests/functional/base.py (hunk 4 above)
Comment (esikachev) [inline my_dev/tests/functional/base.py]: Usually, 'create' methods not use dicts as arguments, this is not clear. If you would like to use dicts we need to refactor all methods and tests in both repos
